docs/howto/account-billing/pull-invoice-data.md

```diff
@@ -0,0 +1,65 @@
+# Pulling invoice data
+
+You may download some or all of your {{brand}} invoices using the
+{{rest_api}}.
```

Comment:
Either include information of how to download the actual (PDF) *invoices* in this how-to, or else change "invoices" to "invoice data" or "invoice information".

Reply:
Makes much more sense, yes.